src/interfaces/token.spec.ts

```diff
@@ -263,40 +266,110 @@ describe("TokenInterface", () => {
       })
     );
 
-    ([250, 42161] as ChainId[]).forEach(chainId =>
-      describe(`when chainId is ${chainId}`, () => {
-        beforeEach(() => {
-          tokenInterface = new TokenInterface(mockedYearn, chainId, new Context({}));
-          tokenInterface.supported = jest.fn().mockResolvedValue([vaultToken, vaultTokenNoBalance, ironBankToken]);
-        });
+    describe("when chainId is 250", () => {
+      beforeEach(() => {
+        tokenInterface = new TokenInterface(mockedYearn, 250, new Context({}));
+        tokenInterface.supported = jest.fn().mockResolvedValue([vaultToken, vaultTokenNoBalance, ironBankToken]);
+      });
 
-        it("should return balances for all supported tokens", async () => {
-          vaultsBalancesMock.mockResolvedValue([vaultTokenWithBalance, vaultTokenWithoutBalance]);
-          ironBankBalancesMock.mockResolvedValue([ironBankTokenWithBalance]);
+      it("should return balances for all supported tokens", async () => {
+        vaultsBalancesMock.mockResolvedValue([vaultTokenWithBalance, vaultTokenWithoutBalance]);
+        ironBankBalancesMock.mockResolvedValue([ironBankTokenWithBalance]);
 
-          const actualBalances = await tokenInterface.balances("0xAccount");
+        const actualBalances = await tokenInterface.balances("0xAccount");
 
-          expect(actualBalances.length).toEqual(2);
-          expect(actualBalances).toEqual(expect.arrayContaining([vaultTokenWithBalance, ironBankTokenWithBalance]));
-          expect(zapperBalancesMock).not.toHaveBeenCalled();
-          expect(vaultsBalancesMock).toHaveBeenCalledWith("0xAccount");
-          expect(ironBankBalancesMock).toHaveBeenCalledWith("0xAccount");
-        });
+        expect(actualBalances.length).toEqual(3);
+        expect(actualBalances).toEqual(
+          expect.arrayContaining([
+            vaultTokenWithBalance,
+            ironBankTokenWithBalance,
+            {
+              address: "0xAccount",
+              balance: "42000000",
+              balanceUsdc: "42000000000000",
+              priceUsdc: "1000000", // $1
```

Comment:
```suggestion
              balance: "42000000000000000000",
              balanceUsdc: "42000000",
              priceUsdc: "1000000", // $1
```
For it to be a more real example, lets use a balance of 42 FTM (Decimal: 18) at a price of 1 USDC (Decimal: 6), results in a USDC balance of 42 USDC (Decimal: 6)

Reply:
Done with https://github.com/yearn/yearn-sdk/pull/262/commits/7d0806ba67870cc56102db636208b9622214c883